Godfathers Armory

AllCorruptingWyrm opened the pull request:
## About The Pull Request

Removes a ROUND START AMERICAN 180
Removes the P90
Removes Combat Medic loadout
Configures out Den loadouts, gives the boss themed loadouts
Den Boss gets a pair (two) of fancy sprited .357 revolvers with tycho
Den get a funny vendor for trades
Den Boss gets a tommy gun for aesthetic

## Why It's Good For The Game

Gangtool was broken so a replacement was made
The P90 is absurd as is the A180
Den Boss should get a tommy gun loadout for aesthetic purposes.

## Pre-Merge Checklist
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog

:cl:
add: Added new things
add: Added more things
tweak: tweaked a few things
/:cl:

panzerr1944 commented on code/modules/WVM/wvm.dm:
```diff
@@ -589,6 +589,41 @@ GLOBAL_VAR_INIT(vendor_cash, 0)
+		new /datum/data/wasteland_equipment("Stinger Grenade",				/obj/item/grenade/f13/stinger,										35),
+		new /datum/data/wasteland_equipment("HE Grenade",					/obj/item/grenade/syndieminibomb/concussion, 						50),
+		new /datum/data/wasteland_equipment("Frag Grenade",					/obj/item/grenade/syndieminibomb/concussion/frag, 					35),
+		new /datum/data/wasteland_equipment("C4",							/obj/item/grenade/plastic/c4, 										85),
```
why are all these so hilariously cheap to the point where any shitter can spawn in and buy one near instantly

AllCorruptingWyrm replied:
> why are all these so hilariously cheap to the point where any shitter can spawn in and buy one near instantly

These were set to this for testing, the outlaw gangtool explosives are also being looked at because they're a bit low, nerfing the low costs of the gangtool is absurd

also note this has been a draft for the last half a day or so, so don't expect it to be as intended.

Keksk commented:
I don't think having the ninja throwing stars inside the vendor is good idea, given the fact they have an embed chance of 100 and fall chance 0 as well as a throw force of 20.

AllCorruptingWyrm commented:
> I don't think having the ninja throwing stars inside the vendor is good idea, given the fact they have an embed chance of 100 and fall chance 0 as well as a throw force of 20.

The gangtool has them for less (30 caps), touching that would mean an out-of-scope PR.

AllCorruptingWyrm commented:
anyway i'm letting this pr brew for the time being as I'd like to collect thoughts, if someone wants to nerf throwing stars than be my guest.

Keksk commented:
> The gangtool has them for less (30 caps), touching that would mean an out-of-scope PR.

The gangtool has the /obj/item/throwing_star and not /obj/item/throwing_star/ninja, which are to be fair just as awful.
I'd just suggest removing them from the vendor and tool.

Serxule commented:
pretty sure the p90 doesnt spawn normally, i've only seen it from admins, and its funny csgo weapon, removing it is stupid. the 180 isnt even good, its literally a bb gun with a large mag. combat medic as in the enclave specialist loadout? why?

BadAtThisGame302 commented:
> pretty sure the p90 doesnt spawn normally, i've only seen it from admins, and its funny csgo weapon, removing it is stupid. the 180 isnt even good, its literally a bb gun with a large mag. combat medic as in the enclave specialist loadout? why?

P90 is OP, Den Boss should not spawn with it.
The 180 isn't good I agree, a 9mm SMG would probably be better.
And they mean the Den Doctor combat medic loadout.

Serxule commented:
> P90 is OP, Den Boss should not spawn with it. 

oh god yeah the p90 should not spawn with den boss, (since im pretty sure its admin-only) if they mean removing it from the loadout then yes do that, but the way they worded it is "removing it from the game"

BadAtThisGame302 commented:
> > P90 is OP, Den Boss should not spawn with it.
> 
> oh god yeah the p90 should not spawn with den boss, (since im pretty sure its admin-only) if they mean removing it from the loadout then yes do that, but the way they worded it is "removing it from the game" (same with combat medic, this is really vague and shit wording)

Yeah it's just being removed from the Den Boss, not from the game.

AllCorruptingWyrm commented:
Did a mapping commit(s) to dumb down spawns, added the vendor I made in this PR, removed all the other funny vendors
what's next is the mapping PR to squish down the den, all done in preparation for said mapping PR.
